tenseal/sealapi/sealapi_util_namespace.cpp

```diff
@@ -152,45 +153,65 @@ void bind_util_namespace(pybind11::module &m) {
              py::arg(), py::arg(), py::arg(),
              py::arg() = MemoryManager::GetPool())
         .def("divide_and_round_q_last_inplace",
-             [](const RNSTool &obj, std::uint64_t *input) {
-                 obj.divide_and_round_q_last_inplace(input,
-                                                     MemoryManager::GetPool());
+             [](const RNSTool &obj, std::vector<uint64_t> &input,
+                std::size_t poly_modulus_degree) {
+                 obj.divide_and_round_q_last_inplace(
+                     RNSIter(input.data(), poly_modulus_degree),
+                     MemoryManager::GetPool());
              })
         .def("divide_and_round_q_last_ntt_inplace",
-             [](const RNSTool &obj, std::uint64_t *input,
+             [](const RNSTool &obj, std::vector<uint64_t> &input,
+                std::size_t poly_modulus_degree,
                 const NTTTables *rns_ntt_tables) {
                  obj.divide_and_round_q_last_ntt_inplace(
-                     input, rns_ntt_tables, MemoryManager::GetPool());
+                     RNSIter(input.data(), poly_modulus_degree), rns_ntt_tables,
+                     MemoryManager::GetPool());
              })
         .def("fastbconv_sk",
-             [](const RNSTool &obj, const std::uint64_t *input,
-                std::uint64_t *destination) {
-                 obj.fastbconv_sk(input, destination, MemoryManager::GetPool());
+             [](const RNSTool &obj, const std::vector<uint64_t> &input,
+                std::vector<uint64_t> &destination,
+                std::size_t poly_modulus_degree) {
+                 obj.fastbconv_sk(
+                     ConstRNSIter(input.data(), poly_modulus_degree),
+                     RNSIter(destination.data(), poly_modulus_degree),
```

Comment:
Shouldn't destination vector be resized to input.size()?

now, by default, it will be empty. isn't that going to lead to a crash?